### DISCUSSION TITLE
Notify doesn't ask me permission to access my media files but it can read them

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#126] [General]
Android 13 splits the "files" permission in "photo" and "audio" files or "access to all files" if you allow "all".

Notify accesses to my gallery without ask me nothing by circumvent permissions pattern.

This is not the right way: control and good behaviors is the way.

Please add both permissions in next version.

This screenshot has taken from Forkgram. This is the proper approach: asking permission for anything.

Power is nothing without control.

![268528929-3ae0cb7d-28a1-4d6a-be6d-cad2e7147261](https://github.com/aritra-tech/Notify/assets/113377431/ada7d06c-63f2-42f7-9fec-d695210d52cd)

**Comment by aritra-tech** [comment]
Got your point.
I will look into it.
